core/templates/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.directive.html

```diff
@@ -30,19 +30,21 @@ <h3>Generate dummy explorations</h3>
       Use this form to generate dummy explorations with random titles.
     </p>
     <span class="row">
-      <label for="explorations to generate">Number of explorations to generate</label>
+      <label for="explorations-to-generate">Number of explorations to generate</label>
       <input class="form-control form-inline" style="margin-bottom: 5px"
              ng-model="$ctrl.numDummyExpsToGenerate"
-             placeholder="Explorations to generate" type="number" min="0">
-      <label for="explorations to publish">Number of explorations to publish</label>
+             placeholder="Explorations to generate" type="number" min="0"
+             id = "explorations-to-generate">
```

Comment:
Remove extra whitespace surrounding `=`. Here and below.

Reply:
Done